### PR TITLE
[Feature][Connector] Add iceberg spark batch sink

### PR DESCRIPTION
**Pull request by eye-gu**
#791

### CODE REVIEW

**Comment by eye-gu** [comment]
@wuchunfu Hi. PTAL thx

File: docs/en/spark/configuration/sink-plugins/Iceberg.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+Refer to [iceberg write options](https://iceberg.apache.org/docs/latest/spark-configuration/) for more configurations.
+
+### path
+
+Iceberg table location.
+
+### saveMode
+
```
Comment (wuchunfu) [inline docs/en/spark/configuration/sink-plugins/Iceberg.md:29]: @ououtt `saveMode` I guess it's better to have a default value

Comment (wuchunfu) [reply docs/en/spark/configuration/sink-plugins/Iceberg.md:29]: Is `saveMode` only these two? I think you can write a link that looks for `saveMode` after

Comment (wuchunfu) [reply docs/en/spark/configuration/sink-plugins/Iceberg.md:29]: The `Options` table above is best arranged neatly

Comment (eye-gu) [reply docs/en/spark/configuration/sink-plugins/Iceberg.md:29]: iceberg only support append and overwrite. and I have set the default value of saveMode to append and format the options table.

**Review by wuchunfu — APPROVED** [review]
LGTM

**Comment by liujinhui1994** [comment]
@ououtt  @wuchunfu  Seatunnel is currently using spark2.4
Will this PR work? According to my understanding iceberg can only read and write existing tables in this version

**Comment by eye-gu** [comment]
> @ououtt @wuchunfu Seatunnel is currently using spark2.4 Will this PR work? According to my understanding iceberg can only read and write existing tables in this version

Yes, the table needs to be created in advance. In my local based on hdfs  test is no problem.

**Comment by liujinhui1994** [comment]
@ououtt  If the table needs to be created ahead of time, I understand that seatunnel-iceberg is invalid?
I feel the need to add the api to create the table in the code.
At least sink should be like this?

**Comment by eye-gu** [comment]
> @ououtt If the table needs to be created ahead of time, I understand that seatunnel-iceberg is invalid? I feel the need to add the api to create the table in the code. At least sink should be like this?

Sorry, I didn't know it was necessary to create a table.

**Pull request merged**